app/sequencescape/sequencescape/api/v2/base.rb

```diff
@@ -2,8 +2,17 @@
 
 class Sequencescape::Api::V2::Base < JsonApiClient::Resource # rubocop:todo Style/Documentation
   class_attribute :plate, :tube
+
+  # Adjusts the parameters used for pagination. We create a custom
+  # class to avoid mutating the global JsonApiClient::Paginating::Paginator object
+  class SequencescapePaginator < JsonApiClient::Paginating::Paginator
+    self.page_param = 'number'
+    self.per_page_param = 'size'
+  end
+
```

Comment:
I noticed .last was pulling down *All* tubes when debugging. It was because we were generating the wrong pagination strings for Sequencescape. Not sure how much of a problem it is in reality (we don't often use last) but this should fix it.